app/services/reports/acquisition_report_service.rb

```diff
@@ -79,18 +103,40 @@ def disposable_diapers_from_virtual_drives
                                                   .sum(:quantity)
     end
 
+    # @return [Integer]
+    def cloth_diapers_from_virtual_drives
+      @cloth_diapers_from_virtual_drives ||= virtual_product_drives
+                                                  .joins(donations: { line_items: :item })
+                                                  .merge(Item.cloth_diapers)
+                                                  .sum(:quantity)
+    end
+
     # @return [Float]
     def percent_donated
       return 0.0 if total_diapers.zero?
 
-      (donated_diapers / total_diapers.to_f) * 100
+      ((donated_disposable_diapers + donated_cloth_diapers) / total_diapers.to_f) * 100
     end
 
     # @return [Float]
-    def percent_bought
+    def percent_purchased
       return 0.0 if total_diapers.zero?
 
-      (purchased_diapers / total_diapers.to_f) * 100
+      ((purchased_cloth_diapers + purchased_disposable_diapers) / total_diapers.to_f) * 100
+    end
+
+    # @return [Float]
+    def percent_cloth_diapers_purchased
+      return 0.0 if purchased_cloth_diapers.zero?
+
+      (purchased_cloth_diapers / total_diapers.to_f) * 100
```

Comment:
This is comparing the purchased number against the number of total diapers, not total cloth diapers. This doesn't seem right?

Reply:
Ahh, i can fix that to compare to the total number of cloth diapers. That makes sense, I think I was thinking of percent of cloth diapers in the whole of inventory. I can def fix this!